edx_oauth2_provider/forms.py

```diff
@@ -88,7 +90,8 @@ def clean(self):
             # verified_email, we can uncomment the following line.
             # or not user.is_active
         ):
-            raise OAuthValidationError({'error': 'invalid_grant'})
+            log.exception("Invalid credentials given with username '{}'.".format(username))
```

Comment:
This error case can happen for multiple reasons.  Not just for invalid credentials.  It can also happen if user doesn't exist.